src/lib/components/icons/apple-icon-light.svelte

```diff
@@ -0,0 +1,18 @@
+<script lang="ts">
+  export let classes = '';
+  export let size = '24px';
+</script>
+
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  height={size}
+  width={size}
+  class={classes}
+  viewBox="0 0 20 20"
+>
+  <path
+    fill="#fff"
+    d="M11.182.008C11.148-.03 9.923.023 8.857 1.18c-1.066 1.156-.902 2.482-.878 2.516.024.034 1.52.087 2.475-1.258.955-1.345.762-2.391.728-2.43zm3.314 11.733c-.048-.096-2.325-1.234-2.113-3.422.212-2.189 1.675-2.789 1.698-2.854.023-.065-.597-.79-1.254-1.157a3.692 3.692 0 0 0-1.563-.434c-.108-.003-.483-.095-1.254.116-.508.139-1.653.589-1.968.607-.316.018-1.256-.522-2.267-.665-.647-.125-1.333.131-1.824.328-.49.196-1.422.754-2.074 2.237-.652 1.482-.311 3.83-.067 4.56.244.729.625 1.924 1.273 2.796.576.984 1.34 1.667 1.659 1.899.319.232 1.219.386 1.843.067.502-.308 1.408-.485 1.766-.472.357.013 1.061.154 1.782.539.571.197 1.111.115 1.652-.105.541-.221 1.324-1.059 2.238-2.758.347-.79.505-1.217.473-1.282z"
+  />
+  <title><slot /></title>
+</svg>
```

Comment:
I added a "light" and "dark" version of the Apple logo given it's the only one that needs to change depending on which mode we're in.

Reply:
Is the shape of the icon actually different? Or, are you just changing the color? I'd personally say all of the icons need to have the appropriate color for the light and dark theme. Are the other icons multi-color? Is that why Apple is different as it is a single color?

Reply:
Yes, we are just changing the colour depending on whether it is included within a light or dark button. Given that the Apple icon is either black or light grey, it felt right to include both and select the right one based on the preferred mode. Facebook and Google however seem to work on both light and dark backgrounds, given they are multi-color.

Reply:
Okay, got it. When looking at this as an isolated component, I'd recommend we don't duplicate two components for the sake of colors. We should pass assign the color via CSS in the way all the icons are assigned colors: https://github.com/cerebrl/forgerock-web-login-framework/blob/main/themes/default/primitives.cjs#L34. Just make sure to remove to assign `none` to the `fill` attribute: https://github.com/cerebrl/forgerock-web-login-framework/blob/main/src/lib/components/icons/eye-icon.svelte#L15.

That way, you can have a light and dark variant without having to duplicate code. I'll continue this thread on a different comment as I'd like to jump into how we know if something needs to be light or dark.

Reply:
I'm looking at bootstrap SVGs now and using the button text color to inform the color of the icon - if I can get this to work then provided plain color icons are within guidelines then I think I can get rid of the dark / light apple logo.

Reply:
We now have just the one Apple icon which takes the color of the text. Not sure what the difference is for the `fill` attribute - there is no visibile difference between `currentColor` which is set now, and `none` which you suggested. Does this need to be changed?